guides/common/modules/proc_creating-a-vmware-user.adoc

```diff
@@ -15,3 +13,8 @@ For VMware vCenter Server version 8.0 or 7.0, set the following permissions:
 * All Privileges -> Virtual Machine -> Edit Inventory (All)
 * All Privileges -> Virtual Machine -> Provisioning (All)
 * All Privileges -> Virtual Machine -> Guest Operations (All)
+
+Additionally, if you want to create virtual machines with a Virtual Trusted Platform Module (TPM) for enhanced security, set the following privileges:
+
+* All Privileges -> Cryptographic operations -> Clone, Encrypt, Encrypt new, Migrate, Register VM
+* All Privileges -> Cryptographic operations -> Direct Access {endash} required to open a console session
```

Comment:
@Gauravtalreja1 have you tried this step in your testing? If so, could you ack this change?

Reply:
yes, this works, @Lennonka just one question what is `{endash}` here?

Reply:
[En dash is a mid-length "hyphen"](https://en.wikipedia.org/wiki/Dash#En_dash). It's just punctuation formatting.